image_provisioner/playbooks/roles/pbench-kickstart/tasks/main.yml

```diff
@@ -1,19 +1,21 @@
 ---
-- name: create directory to store pbench results, store openshift templates for pbench
-  file: 
-    path: "{{ item }}"
-    state: directory
-  with_items:
-     - /var/lib/pbench-agent
-     - /var/home/openshift-templates/pbench
+- block:
+    - name: create directory to store pbench results, store openshift templates for pbench
+      file: 
+        path: "{{ item }}"
+        state: directory
+      with_items:
+         - /var/lib/pbench-agent
+         - /var/home/openshift-templates/pbench
 
-- name: copy openshift template for pbench
-  copy: 
-    src: pbench-agent-daemonset.yml  
-    dest: /var/home/openshift-templates/pbench 
-    owner: root 
-    group: root 
-    mode: 0644
+    - name: copy openshift template for pbench
+      copy: 
+        src: pbench-agent-daemonset.yml  
+        dest: /var/home/openshift-templates/pbench 
+        owner: root 
+        group: root 
+        mode: 0644i
```

Comment:
Eh I think this might be a typo?

Reply:
@sjug good catch, will fix it in one of the existing PR.